domains/shakhzodbek.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "shaxCoder",
+           "email": "shakhgamer1128@gmail.com",
+           "discord": "919229559533097060"
+        },
+    
+        "record": {
+            "CNAME": ["t.me/pyCodeWiz"]
```

Comment:
cant put CNAME in a array, also thats a telegram link so assuming you want domain to redirect it use URL record

Reply:
And CNAME can't use a /